platform/pensando/sonic-platform-modules-dpu/sonic_platform/sfp.py

```diff
@@ -0,0 +1,1875 @@
+#!/usr/bin/env python
+
+# {C} Copyright 2023 AMD Systems Inc. All rights reserved
+#############################################################################
+# Pensando
+#
+# Sfp contains an implementation of SONiC Platform Base API and
+# provides the sfp status which are available in the platform
+#
+#############################################################################
+
+import time
+import struct
+
+try:
+    from sonic_platform_base.sfp_base import SfpBase
+    from sonic_platform_base.sonic_sfp.sff8472 import sff8472InterfaceId
+    from sonic_platform_base.sonic_sfp.sff8472 import sff8472Dom
+    from sonic_platform_base.sonic_sfp.sff8436 import sff8436InterfaceId
+    from sonic_platform_base.sonic_sfp.sff8436 import sff8436Dom
+    from sonic_platform_base.sonic_sfp.inf8628 import inf8628InterfaceId
+    from sonic_platform_base.sonic_sfp.qsfp_dd import qsfp_dd_InterfaceId
+    from sonic_platform_base.sonic_sfp.qsfp_dd import qsfp_dd_Dom
+    from sonic_platform_base.sonic_sfp.sfputilhelper import SfpUtilHelper
+    from sonic_platform.helper import APIHelper
+    import syslog
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
```

Comment:
@shanshri please use SfpOptoe as these are deprecated. Example https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-dell/s6100/sonic_platform/sfp.py

Reply:
@prgeor, addressed it.